src/build/package_compilation_tests.rs

```diff
@@ -1252,6 +1254,7 @@ fn config_compilation_test() {
         vec![],
         vec![OutputFile {
             text: r#"{application, the_package, [
+    {vsn, "1.0.0"}
```

Comment:
@lpil - I am a bit confused why this doesn't need a comma after it?

Reply:
well spotted, it needs a comma. Not sure why it is missing

Reply:
The tests are passing with it and there are other examples in the file that have never had a comma. Maybe make a new ticket for it? I tried to look into it but I couldn't figure out how that string is assembled.

Reply:
Found it. I'll try to fix it up.

Reply:
Done. I think.